### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by mmarchois**
PR 19 

See https://github.com/poledev/mathieu/issues/5

### CODE REVIEW

File: src/SensioLabs/JobBoardBundle/Resources/translations/twig.en.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    my_jobs: My jobs
+    update_job: Update a job
+    preview: Preview
+    publish: Publish
+    save: Save
+    post_job: Post a job
+    make_changes: Make changes
+    how_to_apply: How to apply?
+home:
+    intro: The 1<sup>st</sup> jobboard 100% devoted to the <strong>Symfony</strong> community.<br /><strong>Recruiters</strong> The definitive source for Symfony talent.<br /> <strong>Developers</strong> Find the project, the company, or the job of your dreams.
```
Comment (ogizanagi) [inline src/SensioLabs/JobBoardBundle/Resources/translations/twig.en.yml]: You can improve readability by using this syntax:
```yaml
home:
    intro: > 
        The 1<sup>st</sup> jobboard 100% devoted to the <strong>Symfony</strong> community.<br />
        <strong>Recruiters</strong> The definitive source for Symfony talent.<br /> 
        <strong>Developers</strong> Find the project, the company, or the job of your dreams.
```

http://symfony.com/doc/current/components/yaml/yaml_format.html#strings

File: src/SensioLabs/JobBoardBundle/Resources/views/Base/manage.html.twig [hunk 2]
```diff
@@ -43,12 +43,12 @@
                             </td>
 
                             <td class="action">
-                                <a href="{{ job_url(job, 'job_pay') }}">Pay</a> /
+                                <a href="{{ job_url(job, 'job_pay') }}">{{ ('my_jobs.pay')|trans({}, 'twig') }}</a> /
                                 <a href="{{ job_url(job, 'job_update') }}">
-                                    Make changes
+                                    {{ ('global.make_changes')|trans({}, 'twig') }}
```
Comment (ogizanagi) [inline src/SensioLabs/JobBoardBundle/Resources/views/Base/manage.html.twig]: Unnecessary brackets ?

File: src/SensioLabs/JobBoardBundle/Resources/views/Base/manage.html.twig [hunk 3]
```diff
@@ -43,12 +43,12 @@
...
                                 </a> /
                                 <a href="{{ job_url(job, 'job_remove') }}">
-                                    Delete
+                                    {{ ('my_jobs.delete')|trans({}, 'twig') }}
```
Comment (ogizanagi) [inline src/SensioLabs/JobBoardBundle/Resources/views/Base/manage.html.twig]: Same here

File: src/SensioLabs/JobBoardBundle/Resources/views/Base/manage.html.twig [hunk 4]
```diff
@@ -8,21 +8,21 @@
...
+                <h2 class="big-title">{{ 'global.my_jobs'|trans({}, 'twig') }}</h2>
+                <a href="{{ path('job_post') }}" class="add right"><img class="sprite-button-add-focus-grey-bg" src="{{ asset('bundles/sensiolabsjobboard/images/blank.png') }}" alt="+" /> {{ ('global.post_job')|trans({}, 'twig') }} </a>
                 <table class="table table-striped" id="manage">
                     <thead>
                     <tr>
-                        <th class="span5"> Title </th>
-                        <th class="span2"> Time left </th>
-                        <th class="span2"> Displays </th>
-                        <th class="span4"> Actions </th>
+                        <th class="span5"> {{ ('my_jobs.title')|trans({}, 'twig') }} </th>
```
Comment (ogizanagi) [inline src/SensioLabs/JobBoardBundle/Resources/views/Base/manage.html.twig]: And here

File: src/SensioLabs/JobBoardBundle/Resources/views/Base/manage.html.twig [hunk 5]
```diff
@@ -8,21 +8,21 @@
...
+        ('global.my_jobs'|trans({}, 'twig')): path('manage')
     }) }}
 
     {% embed 'SensioLabsJobBoardBundle:Includes:box.html.twig' %}
         {% block box_content %}
             <div class="padding-table">
-                <h2 class="big-title">My jobs</h2>
-                <a href="{{ path('job_post') }}" class="add right"><img class="sprite-button-add-focus-grey-bg" src="{{ asset('bundles/sensiolabsjobboard/images/blank.png') }}" alt="+" /> Post a job </a>
+                <h2 class="big-title">{{ 'global.my_jobs'|trans({}, 'twig') }}</h2>
+                <a href="{{ path('job_post') }}" class="add right"><img class="sprite-button-add-focus-grey-bg" src="{{ asset('bundles/sensiolabsjobboard/images/blank.png') }}" alt="+" /> {{ ('global.post_job')|trans({}, 'twig') }} </a>
```
Comment (ogizanagi) [inline src/SensioLabs/JobBoardBundle/Resources/views/Base/manage.html.twig]: Plus here :)

File: src/SensioLabs/JobBoardBundle/Resources/views/Base/index.html.twig [hunk 6]
```diff
@@ -8,9 +8,7 @@
             <article>
                 <img src="{{ asset('bundles/sensiolabsjobboard/images/home/megaphone.png') }}" alt="Job !" class="megaphone" />
                 <p class="description">
-                    The 1<sup>st</sup> jobboard 100% devoted to the <strong>Symfony</strong> community.<br />
-                    <strong>Recruiters</strong> The definitive source for Symfony talent.<br />
-                    <strong>Developers</strong> Find the project, the company, or the job of your dreams.
+                    {{ 'home.intro'|trans({}, 'twig')|raw }}
```
Comment (salahm) [inline src/SensioLabs/JobBoardBundle/Resources/views/Base/index.html.twig]: i don't think that `twig` is a relevant domain name for translations

File: app/AppKernel.php [hunk 7]
```diff
@@ -21,6 +21,8 @@ public function registerBundles()
             new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle(),
             new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
             new FOS\JsRoutingBundle\FOSJsRoutingBundle(),
+            new JMS\I18nRoutingBundle\JMSI18nRoutingBundle(),
```
Comment (salahm) [inline app/AppKernel.php]: do you really need these bundles ?

Comment (mmarchois) [inline app/AppKernel.php]: Yes, to have the possibility to prefix routes with the locale and to limit default locale (en|fr)

Comment (florianbelhomme) [inline app/AppKernel.php]: The best bundle E-V-E-R

File: src/SensioLabs/JobBoardBundle/Resources/views/Base/index.html.twig (hunk 6 above)
Comment (florianbelhomme) [inline src/SensioLabs/JobBoardBundle/Resources/views/Base/index.html.twig]: +1

**Pull request merged**